build-system/pr-check/utils.js

```diff
@@ -363,6 +374,7 @@ module.exports = {
   downloadBuildOutput,
   downloadDistOutput,
   downloadEsmDistOutput,
+  downloadSxgDistOutput,
```

Comment:
This PR isn't building and uploading the SXG binaries. You'd need to implement `uploadSxgDistOutput` as well, and add a new parallel job to the build stage. You can use the `Module Dist, Bundle Size` job as an example, except I don't think it's necessary to keep track of the bundle size of SXG builds, so you could just call it `SXG Dist`.

Reply:
Please take a look at the updated `sxg-tests.js`. I tried to mimic the `module-dist-bundle-size.js` task, but I'm not 100% sure if it is the correct way to do it.

Reply:
You'll need to upload the sxg output in a new parallel job that's run in the build stage on Travis, not the test stage. Best way is to make a copy of the `Module Dist, Bundle Size` job and call it `SXG Dist` (skip the bundle size part for now). @erwinmombay can help you with this.

Reply:
@rsimha will do. have a meeting with @xiexr151e in a bit so we'll address it. thank you

Reply:
I made a new parallel job in the `.travis.yml`, as well as the corresponding `sxg-dist.js`. Please take a look at it.

Reply:
Reviewed and posted some new comments. Getting closer!